test/aria-practices/apg.spec.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path');
+const fs = require('fs');
+const { createServer } = require('http');
+const express = require('express');
+const chromedriver = require('chromedriver');
+const AxeBuilder = require('@axe-core/webdriverjs');
+const testListen = require('test-listen');
+const { getWebdriver, connectToChromeDriver } = require('./run-server');
+const { assert } = require('chai');
+const globby = require('globby');
+
+describe('aria-practices', function () {
+  const apgPath = path.resolve(__dirname, '../../node_modules/aria-practices/');
```

Comment:
```suggestion
  const apgPath = require.resolve('aria-practices')
```

Reply:
Gd point. I originally started with those in another dir, but this is better.

Reply:
Never mind. I remember why. aria-practices doesn't have a package.json. Node complains about that.